hips/draw/tests/test_simple.py

```diff
@@ -38,7 +38,7 @@ def get_test_tiles(file_format, survey, order, ipix_list):
 @remote_data
 @requires_hips_extra()
 @pytest.mark.parametrize('pars', draw_sky_image_pars)
-def test_draw_sky_image(pars):
+def test_draw_sky_image(pars, tmpdir):
```

Comment:
Accidental change? Remove?

(just passing `tmpdir` has no effect, you would have to use it from within the test function.